ci-operator/config/openshift/prometheus-config-reloader/master.json

```diff
@@ -0,0 +1,36 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "canonical_go_repository": "github.com/openshift/prometheus-operator",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "prometheus-config-reloader",
+      "dockerfile_path": "Dockerfile.prometheus-config-reloader",
```

Comment:
The Prometheus config reloader is build from the same Github repository, but different Dockerfiles. Is this the only needed change?